README.md

```diff
@@ -74,7 +74,7 @@ await novu.trigger('<TRIGGER_NAME>',
   {
     to: {
       subscriberId: '<UNIQUE_IDENTIFIER>',
-      email: 'john@doemail.com',
+      email: ['john1@doemail.com','roger@doemail.com'],
       firstName: 'John',
       lastName: 'Doe',
     },
```

Comment:
@mrajen27 the syntax is a bit different. Could you update it to this:

```
to: [{
      subscriberId: '<UNIQUE_IDENTIFIER>',
      email: 'john1@doemail.com',
      firstName: 'John',
      lastName: 'Doe',
    }]
```

Reply:
updated as per the comments, pls review